deafrica/platform/sandbox_volume_cleanup.py

```diff
@@ -0,0 +1,104 @@
+import boto3
+import click as click
+from datetime import datetime
+from datetime import timedelta
+from deafrica.utils import setup_logging
+from kubernetes import config, client
+
+# Set log level to info
+log = setup_logging()
+
+try:
+    config.load_incluster_config()
+except config.ConfigException:
+    try:
+        config.load_kube_config()
+    except config.ConfigException:
+        log.exception("Could not configure kubernetes python client")
+
+configuration = client.Configuration()
+
+# create an instance of the API class
+k8s_api = client.CoreV1Api(client.ApiClient(configuration))
+k8s_namespace = "sandbox"
+
+ec2_resource = boto3.resource("ec2")
+ct_client = boto3.client("cloudtrail")
+
+time_now = datetime.now()
+time_back = time_now - timedelta(days=90)
+filters = [
+    {
+        "Name": "tag:kubernetes.io/cluster/deafrica-dev-eks",
+        "Values": [
+            "owned",
+        ],
+    },
+    {
+        "Name": "tag:kubernetes.io/created-for/pvc/namespace",
+        "Values": [
+            k8s_namespace,
+        ],
+    },
+]
```

Comment:
Some of these shouldn't be global and should either be set up in the `cli()` function or in the `delete_volumes()` function.

The only things we want out here as globals are fixed items. Dynamic items will be created every time this file is loaded, so if we wanted to use `delete_volumes` in another tool, the logger, the EC2 and other resources would be created each time it's imported.

I think all of this can go in the `delete_volumes` function...